packages/autorest.python/autorest/codegen/models/client.py

```diff
@@ -101,7 +101,7 @@ def _imports_shared(self, async_mode: bool) -> FileImport:
                 "azure.core", self.pipeline_class(async_mode), ImportType.AZURECORE
             )
 
-        for gp in self.parameters:
+        for gp in self.parameters.positional + self.parameters.keyword_only:
```

Comment:
I find that "api-version" will be included if we use `self.parameters`, and it makes we will generate "from typing import Literal" in the client.py. But the "api-version" is not used in client.py, then the "Literal" will be an unused-imports.

Reply:
I don't think we should change it that way, since we get imports for client's through its parameters, we should handle not passing in a Literal import in the parameter's `def imports` method